pipeline/test/transformers/identifiers.test.ts

```diff
@@ -37,14 +37,14 @@ describe("linkedDocumentIdentifiers", () => {
     expect(identifiers).toIncludeSameMembers(["abc", "def"]);
   });
 
-  it("ignores linked documents without data", () => {
+  it("ignores linked documents without data", async () => {
```

Comment:
Why do we need async here? `linkedDocumentIdentifiers` is synchronous

Reply:
Ok, why `'expect' matchers must be awaited or returned`? The one above is not getting eslint's knickers in a twist

Reply:
Fixed by replacing the playwright/recommended rules with jest/recommended